aria/cli/table.py

```diff
@@ -85,7 +85,7 @@ def get_values_per_column(column, row_data):
 
             return val
         else:
-            return defaults[column]
+            return defaults.get(column) if defaults is not None else None
```

Comment:
should instead have a `defaults = defaults or {}` before calling the nested method